src/components/dashboard/DashboardDetail.js

```diff
@@ -118,7 +118,7 @@ class DashboardDetail extends Component {
                         title={<Translate id='authorizedApps.pageTitle' />}
                         to='/authorized-apps'
                         empty={<Translate id='authorizedApps.dashboardNoApps' />}
-                        accessKeys={authorizedApps}
+                        accessKeys={authorizedApps && authorizedApps.filter(key => key.access_key.permission.FunctionCall.receiver_id !== accountId)}
```

Comment:
why is this filter applied? Also if we need to apply it – shouldn't it happen in reducer so that it happens for every consumer of store?

Reply:
It's to filter out the multisig contract from the 'authorized apps' list. I agree, should happen for every consumer of the store.

Reply:
Moving it to reducer